src/users_router.rs

```diff
@@ -335,3 +400,131 @@ describe! setup_tests {
         };
     }
 }
+
+
+describe! login_tests {
+    before_each {
+        use super::super::users_db::{UsersDb, UserBuilder};
+        use iron::prelude::Response;
+        use iron::Headers;
+        #[allow(unused_imports)]
```

Comment:
Please, don't use this. Instead just import whatever is needed in each specific test.